plugins/kotlin/idea/tests/testData/inspectionsLocal/unusedUnaryOperator/moveToPrevLine/bigDecimal.kt.after

```diff
@@ -0,0 +1,6 @@
+// FIX: Move unary operator to previous line
+// WITH_STDLIB
+fun main() {
+    val a = "2".toBigDecimal() -
+    "4".toBigDecimal()
```

Comment:
I think it would be nice to automatically format this line.

Reply:
d1ab4bc2970f6466696a17aee69371dbc8d8f12d